app/lib/submission_builder/ty2024/states/nj/documents/nj_w2.rb

```diff
@@ -0,0 +1,46 @@
+module SubmissionBuilder
+  module Ty2024
+    module States
+      module Nj
+        module Documents
+          class NjW2 < SubmissionBuilder::Document
```

Comment:
[pebble - this looks good, and no change is necessary] It looks like this class largely duplicates the functionality of ReturnW2 and then renames and removes a bunch of nodes. What do you think about having it subclass ReturnW2 and have a `document` method that looks like
```
def document
  super
  rename and remove nodes
end
```
?